addons/mail/static/src/core/common/channel_member_model.js

```diff
@@ -69,12 +87,8 @@ export class ChannelMember extends Record {
 
     onChangeIsThreadDisplayed(isDisplayed) {
         if (!isDisplayed) {
-            return;
-        }
-        if (this.syncNewMessageSeparator) {
-            this.localNewMessageSeparator = this.new_message_separator;
+            this.syncUnread = true;
```

Comment:
Can move this code in the `onUpdate` of `isDisplayed`, there's no point to keep a function when it's only used once